src/patterns/loading/large-loading.stories.ts

```diff
@@ -0,0 +1,200 @@
+import { Component } from "@angular/core";
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+import { GridModule } from "../../grid/grid.module";
+import { UIShellModule } from "../../ui-shell/ui-shell.module";
+import { ProgressIndicatorModule } from "../../progress-indicator/progress-indicator.module";
+import { PlaceholderModule } from "../../placeholder/placeholder.module";
+import { InputModule } from "../../input/input.module";
+import { DropdownModule } from "../../dropdown/dropdown.module";
+import { LoadingModule } from "../../loading/loading.module";
+import { ButtonModule } from "../../button/button.module";
+import {
+	FormGroup,
+	FormControl,
+	Validators,
+	FormsModule,
+	ReactiveFormsModule,
+	AbstractControl } from "@angular/forms";
+
+@Component({
+	selector: "app-sample-large-loading",
+	template: `
+	<div ibmGrid>
+		<div ibmRow class="header">
+			<ibm-header name="Patterns">
+				<ibm-hamburger></ibm-hamburger>
+			</ibm-header>
+		</div>
+		<div ibmRow class="progress-indicator-wrapper">
+			<ibm-progress-indicator [steps]="steps"></ibm-progress-indicator>
+		</div>
+		<div ibmRow class="form">
+			<form [formGroup]="formGroup" (ngSubmit)="onSubmit()">
+				<div class="bx--form-item">
+					<ibm-label
+						helperText="Optional helper text here; if message is more than one line text should wrap (~100 character count maximum)"
+						[invalid]="isInvalid('input')"
+						invalidText="Please enter a response">
+						Text input label
+						<input
+							ibmText
+							formControlName="input"
+							placeholder="Optional placeholder text">
+					</ibm-label>
+				</div>
+				<div class="bx--form-item">
+					<ibm-label
+						helperText="Optional helper text here; if message is more than one line text should wrap (~100 character count maximum)"
+						[invalid]="isInvalid('textArea')"
+						invalidText="Please enter a response">
+						Text input label
+						<textarea
+							ibmTextArea
+							formControlName="textArea"
+							placeholder="Optional placeholder text">
+						</textarea>
+					</ibm-label>
+				</div>
+				<div class="bx--form-item">
+					<div class="dropdown-wrapper">
+						<ibm-dropdown
+							label="Choose one option"
+							[invalid]="isInvalid('dropdown')"
+							invalidText="Please choose an option"
+							placeholder="Select an option"
+							formControlName="dropdown">
+							<ibm-dropdown-list [items]="items"></ibm-dropdown-list>
+						</ibm-dropdown>
+					</div>
+				</div>
+				<div class="bx--form-item">
+					<button
+						class="form-button"
+						ibmButton
+						type="submit">
+						Show Loading
+					</button>
+				</div>
+			</form>
+		</div>
+	</div>
+	<div *ngIf="isActive">
```

Comment:
Does loading need to be in a separate div? Does it need to be `ngIf`ed if it's inactive?
The code says yes.
If it doesn't need to be in the div, it can still be `ngIf`ed directly if needed.